index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Clarity Fitness|About Us</title>
+    <link href="/Users/uc18609/Desktop/Development/style.css" rel="stylesheet" type="text/css">
```

Comment:
This is fine for this assignment, but in the future you can use relative paths if you don't know the final location of your asset.